distributed/scheduler.py

```diff
@@ -7743,7 +7751,7 @@ def check_idle(self) -> int | None:
                     format_time(self.idle_timeout),
                 )
                 self._ongoing_background_tasks.call_soon(self.close)
-        return None
+        return self.idle_since
```

Comment:
@jacobtomlinson I didn't quite understand this `return None`. I'm not really following the whole idea of `idle` as a boolean on the API in https://github.com/dask/distributed/blob/9eb514b5f3178d65c80c0fc70102c6fb0ccdc0b5/distributed/http/scheduler/api.py#L73

What's the use case for an instantaneous definition of idle? To me, it would make more sense to only expose the `idle_since` field in the API. Consumers of the API would decide if that's been long enough for them to consider the cluster "idle". Whenever any activity happens, `idle_since` is reset, so there's no need for any `if idle and now() - idle_since > cutoff`—shouldn't the `if idle` be superfluous?

Reply:
If the scheduler is currently processing tasks then it is not idle. In that situation what should `idle_since` be set to? The current time doesn't make sense because it isn't idle now, it will become idle at some unknown point in the future.